nucypher/policy/conditions/ecdsa.py

```diff
@@ -143,14 +148,21 @@ def __init__(
         message: Any,
         signature: str,
         verifying_key: str,
+        curve: Optional[str] = NIST192p.name,
         condition_type: str = CONDITION_TYPE,
         name: Optional[str] = None,
     ):
+        if curve not in self.SUPPORTED_CURVES:
+            raise InvalidCondition(
+                f"Unsupported curve: {curve}. Supported curves are: {list(self.SUPPORTED_CURVES.keys())}"
+            )
+
```

Comment:
I don't think you need this validation check because the validation of ExecutionCall schema already does this check for you since the underlying curve field in the schema has a validation associated with it.